Dockerfile

```diff
@@ -6,10 +6,11 @@ RUN  apt-get update \
      flex \
      graphviz \
      python3 \
+     python2 \
```

Comment:
Really?

Reply:
Sadly yes, really. Doxygen is unable to process \ref without this.

Reply:
"Progress is not in enhancing what is, but in advancing toward what will be." Khalil Gibran

Glad someone is watching out for it.